Project/Project/Pages/Index.cshtml

```diff
@@ -21,22 +38,25 @@
         </form>
     </div>
 </div>
-
 @if (Model.SearchCompleted)
 {
-    <table class="table">
-        <thead>
+    <table class="table" style="width:120%">
+        <! --- Width has been corrected - -->
+        <thead class="thead-dark">
+            <! --- table header has colored for better understanding - -->
             <tr>
                 <th>
                     @Html.DisplayNameFor(model => model.NobelLaureates.Laureates[0].Firstname)
                 </th>
                 <th>
                     @Html.DisplayNameFor(model => model.NobelLaureates.Laureates[0].Surname)
                 </th>
-                <th>
+                <th width="9%">
+                    <! --- Width has been corrected - -->
                     @Html.DisplayNameFor(model => model.NobelLaureates.Laureates[0].Born)
                 </th>
-                <th>
+                <th width="9%">
+                    <! --- Width has been corrected - -->
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.